main.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './App'
+
+def main
+  app = App.new
+  app.run
```

Comment:
- [ ] Please take a step back and fix this error, because currently, I cannot run your project.
![image](https://user-images.githubusercontent.com/62705947/180233821-5c5834b2-c29c-4324-a2d5-54cfbdec364d.png)